recipes/full_finetune_distributed.py

```diff
@@ -364,8 +364,10 @@ def _setup_data(
                 for single_cfg_dataset in cfg_dataset
```

Comment:
@ebsmothers and I discussed moving this logic to a config util that can instantiate ListConfigs, but longer term this might also be updated with iterable datasets